setup.cfg

```diff
@@ -33,5 +33,22 @@ console_scripts =
     nt = nitransforms.cli:main
 
 [flake8]
```

Comment:
since we plan on porting this over to nibabel, let's revert the style checks to match theirs

Reply:
ugh right. We might want to push for more strict flake8 checks. @effigies ?

Reply:
Let's go-ahead - good syntax should not be a problem when integrating into nibabel.

Reply:
Ah, sorry. I saw this on my notifications and then it disappeared. Yeah, we can improve things in nibabel, but being more strict here can't cause problems.